include/mbedtls/config_psa.h

```diff
@@ -40,6 +40,10 @@
 
 /* Require built-in implementations based on PSA requirements */
 
+/* We need this to have a complete list of requirements
+ * before we deduce what built-ins are required. */
+#include "psa/crypto_adjust_keypair_types.h"
```

Comment:
Not a blocker, but `config_psa.h` is on its way out. It only exists to facilitate the transition with already-written code that isn't merged yet. Please avoid adding new code to it.

Reply:
Ok, thanks for the info, I'll keep that in mind!